types/semantic-release/index.d.ts

```diff
@@ -8,6 +8,23 @@
 /// <reference types="node" />
 
 declare namespace SemanticRelease {
+    /**
+     * While it may be trivial that multiple analyzeCommits (or any
+     * lifecycle plugins) can be defined, it is not that self-evident
+     * that the plugins executed AFTER the first one (for example, the
+     * default one: commit-analyzer) can change the result. This way it
+     * is possible to create more advanced rules or situations, e.g. if
+     * none of the commits would result in new release, then a default
+     * can be defined.
+     *
+     * For this reason, the types of semantic-release's default analyzer,
+     * commit-analyzer, are noted here.
+     *
+     * See [Plugin development](https://semantic-release.gitbook.io/semantic-release/developer-guide/plugin#multiple-analyzecommits-plugins)
+     * for more details.
+     */
```

Comment:
This documentation doesn't describe the type being documented. I can see it has been copied from the semantic-release documentation, but out of context it doesn't make sense to me.

I think this documentation should say something like "A semver release type. See https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-types.js"